architxt/database/export/graph.py

```diff
@@ -76,7 +146,10 @@ def export_group(
     :param session: Neo4j session.
     """
     properties = get_properties(group)
-    session.run(f"MERGE (n:`{group.label.name}` {{ {', '.join(f'`{k}`: {v!r}' for k, v in properties.items())} }})")
+    session.run(
+        f"MERGE (n:`{group.label.name}` {{ _architxt_group_id: '{group.oid}', {', '.join(f'`{k}`: {v!r}' for k, v in properties.items())} }})"
```

Comment:
You should match only on the id. ou can use something like this

```cypher
MERGE (n:Label {_architxt_group_id: $id})
ON CREATE SET n += $data  // Node does not exist, add the properties
ON MATCH SET n += $data   // Node already exist, add missing properties ?
```

Note that the `$id` and `$data` are query parameter so the DBMS can reuse the execution plan